audioplayer.cpp

```diff
@@ -0,0 +1,99 @@
+#include "audioplayer.h"
+
+vector<string> getFileNames(const string& folder) {
+    vector<string> names;
+    DIR* directory = opendir(folder.c_str());
+
+    if (directory == nullptr) {
+        abort();
+    }
+
+    struct dirent* ent;
+    while ((ent = readdir(directory)) != nullptr) {
+        if ((strcmp(ent->d_name, "..")) && (strcmp(ent->d_name, "."))) {
+            names.emplace_back(ent->d_name);
+        }
+    }
+    closedir(directory);
+    return names;
+}
+
+void AudioPlayer::closeAudio() {
+    callMciSendString("close mp3");
+}
+
+void AudioPlayer::playAudio() {
+    // save current volume
+    callMciSendString(("setAudio mp3 volume to " + currentVolume));
+
+    // play audio
+    isPlaying = true;
+    callMciSendString("play mp3");
+}
```

Comment:
What if there's no track selected, how should this method handle this situation?